locustfile.py

```diff
@@ -60,3 +63,14 @@ def idemia_enrollment(self):
             headers={"Authorization": self.bearer_token},
             name="/idemia/enrollment/uuid",
         )
+
+
+class SKTestUser(HttpUser):
```

Comment:
Check `pylint` warnings. We should be adding docstrings to both classes and functions to help with readability. It's not immediately clear what the SKTestUser will do within the `hello_world` (change name?) function, especially since the URL is obfuscated.

Reply:
If you haven't already, for all of your Python repos, set up `pylint` and get set up with the editor integrations here: http://pylint.pycqa.org/en/latest/user_guide/ide-integration.html

Reply:
added doc string.

Reply:
check is failing due to older black version